### DISCUSSION TITLE
Performance Monitoring iOS integration

### DISCUSSION THREAD

**Discussion by olegmarkevychpush** [invertase/react-native-firebase#7442] [Q&A]
Packages info:
`react-native`: 0.65.3
`react-native-firebase` packages: 10.7.0 (for now I'm not able to use newer versions)

I'm trying to implement Performance Monitoring.
For that I am using this guide https://rnfirebase.io/perf/usage.
Firebase is configured correctly because we are using `@react-native-firebase/crashlytics` and `@react-native-firebase/analytics` for years in our project. 
Issue is that on **iOS** I don't receive data in firebase dashboard and it still tells me that I need to add SDK (Android works completely fine).
Question is: do I need to do some extra stuff for v10? Because I did everything from the tutorial and firebase troubleshooting docs.
XCode logs tell me that Firebase/Performance works fine, but still no info in dashboard.
Will be grateful for any help, thanks

**Comment by mikehardy** [comment]
If the logs indicate things are working, but there is nothing on the dashboard, it may be a cloud / console thing and would unfortunately be out of our ability to control here. I have in the past seen a lot of latency with regards to time between data being sent from the app and data showing up on the console, so it could be something like this.

I must take a moment and say that these versions:

> react-native 0.65.3
> react-native-firebase packages: 10.7.0 (for now I'm not able to use newer versions)

...are basically unsupportable at this point, as we need to work with Xcode 15 / iOS 17 and those older versions won't even compile with Xcode 15. I would urge you to prioritize getting up to date, and it may resolve the performance monitoring issue as well...

**Reply by olegmarkevychpush** [reply]
Thanks for your reply, I will try to wait some time. Mb it's issue from firebase side. Also wrote to them about this
